.changeset/red-cobras-sell.md

```diff
@@ -0,0 +1,5 @@
+---
+"@workleap/create-schemas": patch
```

Comment:
Mostly meant to test if changeset works.

We can combine multiple changes in a single release, so this will not bump the version of the package until we decide to release (at which point, I expect other, more meaningful changes to also be included)